package.json

```diff
@@ -102,6 +102,11 @@
                     "default": false,
                     "description": "Whether to automatically apply lint fixes from buildifier when formatting a Bazel file."
                 },
+                "bazel.queriesShareServer": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Use same Bazel server for queries and builds."
```

Comment:
```suggestion
                    "description": "Use the same Bazel server for queries and builds. By default, vscode-bazel uses a separate server for queries so that they can be executed in parallel with builds. You can enable this setting if running multiple Bazel servers has a negative performance impact on your system, but you may experience degraded performance in Visual Studio Code for operations that require queries."
```